jdbc-adapter/virtualschema-jdbc-adapter/src/test/java/com/exasol/adapter/dialects/IntegrationTestSetup.java

```diff
@@ -15,65 +17,142 @@
 import org.apache.http.entity.FileEntity;
 import org.apache.http.impl.client.HttpClientBuilder;
 
-
 /**
- * The main method of this class will be called in the pre-integration-test maven phase before the actual integration-test phase starst.
- * Here we have to setup the integration test environment
+ * The main method of this class will be called in the
+ * <code>pre-integration-test</code> Maven phase before the actual
+ * <code>integration-test</code> phase starts.
+ *
+ * <p>
+ * We upload the JAR file containing the virtual schema adapter to BucketFS so
+ * that we don't accidentally forget to do this before the integration test.
+ * There is a caveat though: if you run this on a cluster, the cluster nodes
+ * will not replicate the JAR instantly. This takes a while. So in clustered
+ * environments it is safer to upload the file and wait a while.
+ * </p>
  *
- * Attention: This does not deploy the latest jar, because the dist maven module is cleaned and build after this module. Right now you need to do something like "mvn clean package && mvn verify -Pit -D..." to upload and test the latest jar.
+ * <p>
+ * <strong>Attention:</strong>
+ * </p>
+ *
+ * <p>
+ * This does not deploy the latest JAR, because the distribution Maven module is
+ * and and build after this module. Right now you need to do something like
```

Comment:
`and and` typo.

Right now I wonder if it is not possible to do it all in one Maven call. Maybe the integration tests can be moved into the pom.xml on highest level and this will be executed at the end? Not sure, ... Anyway, such change is not part of this PR.

Reply:
The integration tests need to be redone from the ground on.
They need to automatically create a clean environment for each test, we need cleaner and most importantly more compact configuration and uniform testing across all dialects. While we are at it we also need to eliminate code duplication.

That is a separate PR for sure.